static/css/restyle.less

```diff
@@ -0,0 +1,853 @@
+    }
+
+    &:after {
+        background: none;
+        color: rgba(255,255,255, 1);
+        height: 100%;
+        border: 0;
+        font-family: 'Open Sans';
+        line-height: 260px;
+        vertical-align: middle;
+        border-radius: 0;
+        box-shadow: none;
+    }
+}
+
+.control {
+    font-family: 'Open Sans' !important;
+}
+
+#promos .prev {
+    left: 0;
+}
+#promos .next {
+    right: 0;
+}
+#promos h2, #promos h3 {
+    font-style: normal;
+    font-family: inherit;
+}
+#promos .promo h2, #promos .ryff h3 {
+    font-size: 36px;
+    font-weight: 100;
+    line-height: 1.3em;
+    color: #fff;
+}
+#promos h3 {
+    color: #fff;
+}
+.promo h2 {
+    text-shadow: none;
+}
+
+
+/* Featured/Monthly Add-on Carousel Element */
+/* move icon to the left */
+#featured-addon .wrap > div,
+#monthly .wrap > div {
+    display: block;
+}
+#featured-addon .blurb,
+#featured-addon .thumb,
+#monthly .blurb,
+#monthly .thumb {
+    display: block;
+}
+#featured-addon .blurb,
+#monthly .blurb {
+    max-width: 80%;
+    float: right;
+}
+#featured-addon .thumb,
+#monthly .thumb {
+    float: left;
+}
+
+/*  */
+#featured-addon > div, #go-mobile > div, #monthly > div, .addons li > a {
+    border: 0;
+    border-radius: 0;
+    box-shadow: none;
+    background: none;
+    background-color: rgba(255, 255, 255, 0.0);
+    transition-property: background-color;
+}
+.addons li > a:active, .addons li > a:focus, .addons li > a:hover {
+    box-shadow: none;
+    background-color: rgba(255, 255, 255, 0.2);
+    transition-property: background-color;
+}
+.addons img {
+    float: left;
+    margin-right: 8px;
+    margin-bottom: 3px;
+}
+.addons h3, #featured-addon h3, #monthly h3 {
+    font-size: 1.2rem;
+    line-height: 1.3rem;
+    max-height: 1.2rem;
+    font-weight: 200;
+    text-align: left;
+    padding-bottom: 5px;
+}
+.addons li > a:active h3,
+.addons li > a:focus h3,
+.addons li > a:hover h3 {
+    text-decoration: none;
+}
+#featured-addon h3 a, #monthly h3 a {
+    color: #fff;
+}
+.addons p.desc, #monthly .wrap p {
+    max-height: 5.4em;
+    font-size: 0.85rem;
+    font-weight: 200;
+    color: #fff;
+    text-overflow: ellipsis;
+    clear: both;
+}
+
+/* Side nav on Home */
+.home #side-nav {
+    /* pushed down to make room for more header */
+    margin-top: 0;
+}
+
+
+/* Install Button */
+.button,
+button,
+input[type="submit"] {
+/* font: 700 13px/1em "Trebuchet MS",sans-serif;*/
+    font-family: inherit;
+    background: #5784BF;
+    border-radius: 0;
+    box-shadow: none;
+    border: 0 none;
+}
+.button.add,
+.button.good,
+button.good {
+    background: #489615;
+}
+.button.selected,
+.button:active,
+.button:focus,
+.button:hover {
+    box-shadow: 0 0 100px rgba(255, 255, 255, 0.2) inset;
+}
+
+
+.secondary {
+    margin: 12px 0;
+
+    h2 {
+        font-family: inherit;
+        font-style: normal;
+        font-weight: 700;
+        text-transform: uppercase;
+        color: #aaa;
+    }
+
+    li {
```

Comment:
@sarahmonster pointed out that removing these borders makes it feel a bit disjointed, and I'm inclined to agree.

<img width="1037" alt="screenshot 2015-12-23 00 13 09" src="https://cloud.githubusercontent.com/assets/90871/11968938/260a30d2-a90a-11e5-82f8-02551e96d9df.png">
vs
<img width="999" alt="screenshot 2015-12-23 00 13 02" src="https://cloud.githubusercontent.com/assets/90871/11968937/260a2538-a90a-11e5-8b9f-b1b454b68e64.png">

I would keep them in (maybe modify them somehow but not remove them). It helps anchor the sidebars.